docs/source/contributing/walkthrough.md

```diff
@@ -80,7 +81,7 @@ GitHub homepage
 1. Once you are logged in, navigate to the [SATRE Specification page](https://github.com/sa-tre/satre-specification)
 
 2. This is the SATRE specification **repository**. You can think of this like a directory containing all files to do with the SATRE specification.
-   The main things to know about are the link in the `About` Section, and the `Code`, `Issues` and `Pull requests` tabs.
+   The main things to know about are the link in the Section, and the `Code`, `Issues` and `Pull requests` tabs.
```

Comment:
Not sure why you deleted `About`? Isn't it it referring to the link to the website in the `About` section?

Reply:
Auto-formatter error. It interprets `text` as whitespace and so wants to replace " `About` " with " ". Good catch.